src/linear_algebra/general_linear_group.lean

```diff
@@ -0,0 +1,199 @@
+/-
+Copyright (c) 2020 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+
+The General Linear group $GL(n, R)$
+-/
+import linear_algebra.matrix
+import linear_algebra.matrix.nonsingular_inverse
+import linear_algebra.special_linear_group
+import linear_algebra.determinant
+
+
+/-!
+# The General Linear group $GL(n, R)$
+
+This file defines the elements of the General Linear group `general_linear_group n R`,
+consisting of all invertible `n` by `n` `R`-matrices.
+
+
+## Main definitions
+
+ * `matrix.general_linear_group` is the type of matrices over R with unit determinant
+ * `matrix.GL.group` gives the group structure (under multiplication)
```

Comment:
```suggestion
```
This doesn't exist any more, the group structure is just `units.group`

Reply:
done